spec/spec_helper.cr

```diff
@@ -23,7 +23,6 @@ Avram.initialize_logging
 
 Db::Create.new(quiet: true).run_task
 Db::Migrate.new(quiet: true).run_task
-Db::VerifyConnection.new(quiet: true).run_task
```

Comment:
This isn't needed. We test this both in the spec, and through the integration test script on CI.